node/modules/lp2p/pubsub.go

```diff
@@ -56,6 +89,11 @@ func GossipSub(mctx helpers.MetricsCtx, lc fx.Lifecycle, host host.Host, nn dtyp
 						return 2500
 					}
 
+					_, ok = drandBootstrappers[p]
+					if ok && !isBootstrapNode {
```

Comment:
the `!isBootstrapNode` check seems unnecessary, we just checked this above and would have returned 2500 already.

Reply:
Actually never mind, it's a check of whether the node is an actual bootstrap node, so it's correct.